addons/web/static/src/views/fields/relational_utils.js

```diff
@@ -564,6 +564,7 @@ X2ManyFieldDialog.props = {
     save: Function,
```

Comment:
Can you rephrase the commit message, a bit like I suggested in the other PR?

One giant sentence is very hard to follow, split that into multiple sentences. 

I will also explain that it only changes the button label, right now when reading it feels like you are changing the behavior.

Reply:
Looking better, thanks !

Can we just alter a bit the header as well?

Instead of:
> [FIX] web: ux change in field dialog footer

Would go for:
> [IMP] web: adapt the kanban "Remove" button label